test/gtest/spmv_test.cpp

```diff
@@ -1,25 +1,36 @@
 #include <gtest/gtest.h>
 
```

Comment:
For testing, I support the use of rocThrust suggested by @BenBrock . For implementation, I don't ;)